contract_compassion/models/contracts.py

```diff
@@ -215,6 +218,18 @@ def _compute_contract_duration(self):
                 contract.contract_duration = (
                     end_date - contract_start_date).days
 
+    def _compute_last_letter(self):
+        """
+        Get the date of the last letter sent to the child by the sponsor and
+        compute since how many days he didn't write him any letter
+        :return: int value
+        """
+        self.last_letter = (date.today() - datetime.strptime(
```

Comment:
The compute method should always be able to be invoked on a recordset. This means that you should not compute only for one record but put a for loop and setting the values on each record:
```
for contract in self:
    contract.last_letter = ....
```